momaland/learning/discrete/cooperative_momappo.py

```diff
@@ -316,7 +318,8 @@ def _update_epoch(update_state, unused):
 
 def train(args, env, weights: np.ndarray, key: chex.PRNGKey):
     """MAPPO scalarizing the vector reward using weights and weighted sum."""
-    num_updates = int(args.total_timesteps // args.num_steps)
+    # num_updates = int(args.total_timesteps // args.num_steps)
```

Comment:
?

Reply:
my bad, that was for testing for things to go quickly...